plugins/onex/skills/deploy_local_plugin/deploy.sh

```
@@ -1324,6 +1324,62 @@ if [[ "$EXECUTE" == "true" ]]; then
+        fi
+    done
+    if [[ "$AUTHORITY_FAILED" == "true" ]]; then
+        echo -e "${RED}  Runtime authority assertion FAILED — hooks resolve outside canonical cache${NC}"
+        exit 1
+    fi
+    echo -e "${GREEN}  Runtime authority verified: all hooks resolve through canonical cache${NC}"
+
+    # Runtime execution proof: invoke session-start.sh and verify it executes [OMN-7017]
+    SMOKE_HOOK="$CANONICAL_PATH/hooks/scripts/session-start.sh"
+    if [[ -x "$SMOKE_HOOK" ]]; then
+        SMOKE_OUTPUT=$(echo '{"sessionId":"deploy-smoke","projectPath":"/tmp","cwd":"/tmp"}' | bash "$SMOKE_HOOK" 2>/dev/null || true)
```

Shellcheck (0.11.0)

[warning] SMOKE_OUTPUT appears unused. Verify use (or export if used externally).

(SC2034)